### PR TITLE
Garage App V1

### PR DESCRIPTION
**Pull request by sanderpick**

### CODE REVIEW

**Review by dtbuchholz — APPROVED** [review]
👍

File: garage/src/utils/async.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  const newOpts = {
+    initialDelay: wait,
+    wait,
+    maxNumberOfAttempts: maxNumberOfAttempts--,
+    onMaxNumberOfAttemptsReached,
+  };
+
+  const cancelled = isCancelled ? isCancelled() : false;
```
[the remark this answers is not in the document]
Comment (datadanne) [reply garage/src/utils/async.ts:48]: oops

**Review by datadanne — APPROVED** [review]
LGTM 😉

File: garage/src/utils/async.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanderpick) [reply garage/src/utils/async.ts:48]: it's technically a correct spelling so just resolved this one haha

**Pull request merged**